Source/Modules/python.cxx

```diff
@@ -922,15 +943,15 @@ class PYTHON:public Language {
    * as a replacement of new.instancemethod in Python 3.
    * ------------------------------------------------------------ */
   int add_pyinstancemethod_new() {
-    String *name = NewString("SWIG_PyInstanceMethod_New");
-    String *line = NewString("");
-    Printf(line, "\t { \"%s\", %s, METH_O, NULL},\n", name, name);
-    Append(methods, line);
     if (fastproxy) {
+      String *name = NewString("SWIG_PyInstanceMethod_New");
+      String *line = NewString("");
+      Printf(line, "\t { \"%s\", %s, METH_O, NULL},\n", name, name);
+      Append(methods, line);
       Append(methods_proxydocs, line);
+      Delete(line);
+      Delete(name);
     }
-    Delete(line);
-    Delete(name);
```

Comment:
What's going on here?

You said "This PR doesn't change anything for the existing use cases" but this seems to change ... something ... to be only done under `fastproxy` outside of the limited ABI case.

Reply:
Looks like a mistake in the [original changes](https://github.com/swig/swig/pull/1613/files#diff-c686001fe7d4424a7542f21deceb3f4c8e5dc806ec3edbf9cb36444747456a8b), thanks for noticing it!

Fixed now.

Reply:
No, sorry, it's my mistake. `SWIG_PyInstanceMethod_New()` is only used with "fastproxy" and we must not emit it when not using it as it wouldn't even compile with limited API.

I.e. the previously emitted declarations were actually useless before and this fixes it as a (welcome, even if minor) side effect.

So, finally, there is nothing to fix here.

Reply:
I've opened #2191 to address this sub-issue more completely - the condition these are actually used under is `!builtin && fastproxy` so I've used that here, for the static equivalent just below, and in `pyinit.swg`.  I did some limited testing locally and it looked good, but CI is much more thorough.  Assuming that can be merged it'll simplify this patch somewhat.  I have a merged version I can push if you want (I did a test merge to see if this actually allowed `-py3-stable-abi -builtin` to work, but it doesn't).

Reply:
#2191 now merged.

Reply:
I'm going to have to rebase and force push now, as this can't be merged any more after this merge...